Quoridor.Model/IPlayer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Quoridor.Model
+{
+    interface IPlayer
+    {
+        Position currentCell { get; }
+        Wall[] Walls { get; }
```

Comment:
by the way, I don't think, that We need **Wall[]** field. Instead of this We can make **int WallsCount** and decrease this number, when placing a wall

Reply:
Agree with @VsIG-official, also we can make it so that in constructors this field has default value 10